app/ts/components/App.tsx

```diff
@@ -58,17 +58,16 @@ export function App() {
 
 	const updateInfos = async () => {
 		try {
-			if (!isValidEnsSubDomain(inputValue.value)) return
-
-			const ensSubDomain = inputValue.value
+			const ensSubDomain = inputValue.value.toLowerCase()
```

Comment:
We may want to consider using the term "name" instead of "domain".  Domain is specific to the DNS system ("domain name system") while ENS is also a name system, it doesn't use the "domain" terminology.